modules/openapi-generator/src/main/resources/Java/libraries/okhttp-gson/ApiClient.mustache

```diff
@@ -187,11 +187,22 @@ public class ApiClient {
     /**
      * Set HTTP client
      *
-     * @param httpClient An instance of OkHttpClient
+     * @param newHttpClient An instance of OkHttpClient
      * @return Api Client
      */
-    public ApiClient setHttpClient(OkHttpClient httpClient) {
-        this.httpClient = httpClient.newBuilder().addInterceptor(getProgressInterceptor()).build();
+     public ApiClient setHttpClient(OkHttpClient newHttpClient) {
```

Comment:
@burberius looks like an additional space was added to this line. Can you please remove it to make the indentation 4-space?

Reply:
Let me fix it in another PR as I want to file a PR to improve the overall indentation

Reply:
FYI. I've filed #2435 to fix the Javadoc issue.